tt_metal/hw/ckernels/wormhole_b0/metal/common/metal_ckernel_globals.h

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
this code came from ckernel_globals but should never have been put in there (had to due to include issues).  it really belongs under hw/inc/<xyz.h>, maybe xyz=firmware_common.